InferenceSystem/deploy-aci-githubaction.md

```diff
@@ -0,0 +1,53 @@
+### Deploy ACI through GitHub Action (Update New AI4Orca Inferencing docker images)
+
+- Ref. Documents
+[Configure a GitHub action to create a container instance](https://docs.microsoft.com/en-us/azure/container-instances/container-instances-github-action#create-service-principal-for-azure-authentication)
+
+After the updated AI4Orca ML infernece docker images are pushed to the ACI registry, the following steps are required to update the ACI deployment.
+
+
+1. __Step 1. Conigure GitHub Secret__
```

Comment:
you can remove "Step 1." since you already have 1.

Reply:
```suggestion
1. __Configure GitHub Secrets__
```